pkg/logcli/client/file.go

```diff
@@ -78,6 +78,9 @@ func (f *FileClient) Query(q string, limit int, t time.Time, direction logproto.
 		uint32(limit),
 		nil,
 	)
+	if err != nil {
+		return nil, err
```

Comment:
Should we wrap these errs, or is it handled neatly by the caller?

Reply:
Yes. The other errors are wrapped. Done.